README.md

```diff
@@ -54,6 +54,24 @@ jaf -configFile example.conf
 ```
 Of course, you can also write a init system script to handle this for you.
 
+### Running from Docker
+Running it from the GitHub Container Registry
+```bash
+docker run \
+    -v /path/to/your/config.conf:/app/jaf.conf \
+    -v /path/to/where/you/want/your/files/to/be/stored:/FileDir/in/your/config \
+    ghcr.io/leon-richardt/jaf:latest
+```
+
+Building the Docker image and running it locally
+```bash
+docker build -t jaf .
+docker run \
+    -v /path/to/your/config.conf:/app/jaf.conf \
+    -v /path/to/where/you/want/your/files/to/be/stored:/FileDir/in/your/config \
+    jaf
+```
```

Comment:
I think it would also be reasonable to mention somewhere that port mappings need to be set up as well if you're not running behind a reverse proxy.

Reply:
Added some documentation for this - I chose a different port on purpose to make them easier to talk about but happy to change